tests/loader.lisp

```diff
@@ -64,11 +80,25 @@
   (string-trim '(#\Space #\Newline)
                (format nil "~{~A~%~}" (reverse lines))))
 
+(defun first-integer (string)
+  "Return the first integer appearing in STRING, or nil."
+  (flet ((numeric? (c)
+           (<= 48 (char-code c) 57)))
+    (let ((start (position-if #'numeric? string)))
+      (when start
+        (let* ((string (subseq string start))
+               (end (position-if (complement #'numeric?) string)))
```

Comment:
This maybe should check for decimal points.

Probably unecessary, if we are diligent about adding numbers to the beginning of tests, but what if the name of the test is something like, `Ensuring proper behavior sbcl >2.4`:

`(first-integer "Ensuring proper behavior sbcl >2.4`")` returns 2.

Reply:
since this is mainly here to facilitate a developer's selection of a single test case, I think it's safe to presume they would assign an unambiguous number to a failing case, via a local edit.

At some point, a stricter header structure could be imposed -- I'm inclined to leave it loosely defined for now.